.github/actions/getReleasePullRequestList/getReleasePullRequestList.js

```diff
@@ -0,0 +1,37 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const GitUtils = require('expensify-common/lib/GitUtils.jsx');
+
+const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
+const inputTag = core.getInput('TAG', {required: true});
+
+console.log('Fetching release list from github...');
+octokit.repos.listTags({
+    owner: github.context.repo.owner,
+    repo: github.context.repo.repo,
+    per_page: 100,
+})
+    .catch(githubError => core.setFailed(githubError))
+    .then((githubResponse) => {
+        const tags = githubResponse.data.map(tag => tag.name);
+        let priorTag;
```

Comment:
This is maybe simpler than the loop you have here:

```suggestion
        const priorTagIndex = _.indexOf(tags, inputTag) - 1;
        if (priorTagIndex < 0) {
            core.setFailed('Given tag is not found in the last 100 release tags.');
            return;
        }
        
        const priorTag = tags[priorTagIndex];
        ...
        ...
```